.github/workflows/generate-dto.yml

```
@@ -0,0 +1,45 @@
+name: Generate DTOs based on JSON Schema
+
+on:
+    push:
+        branches:
+            - main
+        paths:
+            - 'bancos.json'
+    workflow_dispatch:
+
+jobs:
+  Generate-DTOs:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)